### DISCUSSION TITLE
"fatal: Could not read from remote repository" when pushing to Github

### DISCUSSION THREAD

**Discussion by Gabrielle-Tobermann** [nss-evening-cohort-14/evening-client-side#28] [Help Me!] [answered]
Name: Gabby Tobermann 

Description: When trying to push the Javacript exercises to my repo on Github, I get the error: 

The authenticity of host 'github.com (140.82.114.4)' can't be established.
RSA key fingerprint is SHA256:nThbg6kXUpJWGl7E1IGOCspRomTxdCARLviKw6E5SY8.
Are you sure you want to continue connecting (yes/no/[fingerprint])? yes
Warning: Permanently added 'github.com,140.82.114.4' (RSA) to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository. Please make sure you have the correct access rights and the repository exists.

Google links: https://careerkarma.com/blog/git-fatal-could-not-read-from-remote-repository/

This link tells me the problem is with my SSH key. I checked on Github and my SSH key is there. But, the SSH key present in the error message in my terminal is different from my SSH key on Github.

**Comment by drteresavasquez** [comment]
You need to walk through the install steps again with your ssh key. It’s possible something may have gotten mixed up. Do that and let me know if that resolves your issue.

**Reply by drteresavasquez** [reply]
@Gabrielle-Tobermann if it does not work, let me know and I can jump on to make sure you have/get everything setup correctly so you won’t have the issue anymore.

**Reply by Gabrielle-Tobermann** [reply]
I went through instructions and made a new key. It didn't work. error: Warning: Permanently added the RSA host key for IP address '140.82.114.3' to the list of known hosts.
git@github.com: Permission denied (publickey).
fatal: Could not read from remote repository.

**Reply by Gabrielle-Tobermann** [reply]
(Creating the new key worked, I meant pushing to github again didn't work)

**Reply by drteresavasquez** [reply]
For now just do your work locally and run the test locally until they all pass. We can get together before class on Tuesday to get the repo connected. You need to make sure that you have your public key set up properly. This shouldn’t hold up your ability to finish the assignment if you haven’t already finished it.

**Reply by drteresavasquez** [reply]
Also, did you start from the beginning as if you never created a key through the docs? Creating a new key locally, etc?

**Reply by Gabrielle-Tobermann** [reply]
Okay, sounds good. Thank you! Enjoy your weekend :)

**Reply by Gabrielle-Tobermann** [reply]
Yes. I created my key in my terminal just like in the video and then added it to github.

**Comment by drteresavasquez** [comment]
@Gabrielle-Tobermann watch this video to help you complete the assignment without needing the code on your local machine. It is a little more cumbersome, but at least it gets you an option to complete the assignment. We can discuss your issue and take a look on Tuesday.

https://vimeo.com/501565182/59d6c13c20

**Comment by drteresavasquez** [comment, marked as answer]
I am going to close this ticket because it is similar to #30 please continue the conversation there if there are any other additions
